src/definition/uikit/IUIKitSurface.ts

```diff
@@ -17,3 +18,7 @@ export interface IUIKitView {
     clearOnClose?: boolean;
     notifyOnClose?: boolean;
 }
+
+// Added for backwards compatibility
+export type IUIKitView = IUIKitSurface;
+export import UIKitViewType = UIKitSurfaceType;
```

Comment:
These definitions have to be in the `src/definition/uikit/IUIKitView.ts`, as apps could have imported them directly. Removing the file will make them break